napari/plugins/hook_specifications.py

```diff
@@ -352,6 +352,20 @@ def napari_experimental_provide_function_widget() -> Union[
         while the third element should provide keyword arguments for
         :meth:`napari.qt.Window.add_dock_widget` (though note that the
         ``shortcut=`` keyword is not yet supported).
+        
+    Examples
+    --------
+    >>> from magicgui import magicgui
+    >>> from napari_plugin_engine import napari_hook_implementation
+    >>> 
+    >>> @magicgui
```

Comment:
Based on [this conversation on Zulip](https://napari.zulipchat.com/#narrow/stream/212875-general/topic/Making.20my.20first.20napari.20plugin.20.2F.20hook.20implementations/near/222761929), the magicgui decorator should be removed.

```suggestion
    >>> from napari_plugin_engine import napari_hook_implementation
    >>> 
```